.vscode/settings.json

```diff
@@ -1,5 +1,10 @@
 {
   "eslint.validate": ["javascript", "typescript", "vue"],
   "eslint.useFlatConfig": true,
-  "typescript.tsdk": "./frontend/node_modules/typescript/lib"
+  "typescript.tsdk": "./frontend/node_modules/typescript/lib",
+  "python.testing.pytestArgs": [
```

Comment:
Could you give me a quick explanation of why these lines are needed, @zeus2611? Thank you!

Reply:
VS Code automatically added these lines while I was working on the tests. It seems they were included as part of the editor's configuration updates. Let me know if you’d like me to investigate further!

Reply:
Let's remove these then :) Can you do another commit to do this?

Reply:
Ah you already did :)

Reply:
Thank you!